manifests/g/GNU/Solfege/3.23.4/GNU.Solfege.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: GNU.Solfege
+PackageVersion: 3.23.4
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+UpgradeBehavior: install

```

Comment:
Another `ProductCode` so `winget upgrade` and `list` works as expected
Forgot to explain on the last one, but this is required because the Publisher is missing from the ARP
![image](https://user-images.githubusercontent.com/15158490/140578560-9af6f223-6215-4c8a-a680-22ecd8386c32.png)

```suggestion
UpgradeBehavior: uninstallPrevious
ProductCode: 'GNU Solfege_is1'
```

Also moved the UpgradeBehavior to this field